sdk/ml/azure-ai-ml/setup.py

```diff
@@ -85,15 +85,21 @@
         "azure-common<2.0.0,>=1.1",
         "typing-extensions<5.0.0",
         # "opencensus-ext-azure<2.0.0", disabled until SDK logging re-activated
-        # Used in pipeline_component_builder
-        'bytecode<0.15.0,>=0.13.0',
     ],
     extras_require={
         # user can run `pip install azure-ai-ml[designer]` to install mldesigner alone with this package
         # so user can submit @dsl.pipeline with @mldesigner.command_component inside it.
         "designer": [
             "mldesigner",
         ],
+        # user can run `pip install azure-ai-ml[bytecode]` to install bytecode, which is used to accelerate dsl
+        # pipeline builder.
+        "bytecode": [
```

Comment:
Do you foresee we need to add other dependencies that are specific to dsl logic? If yes, I'm thinking we should name the extra to be more generic. E.g. dsl-dependencies or pipeline-builder.

@annatisch please share your thoughts too. Thank you.

Reply:
As I mentioned to @elliotzh in email - this should be removed from setup.py entirely.

Reply:
@annatisch updated